### PR TITLE
Add duythinht's solution

### PR DESCRIPTION
**Pull request by duythinht**

### CODE REVIEW

File: algorithms/solutions/week21/duythinht.solutions.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+### Problem 1
+
+https://leetcode.com/problems/valid-anagram/description/
+
+
+```C
+#define ABC_SIZE 26
```
Comment (qcam) [inline algorithms/solutions/week21/duythinht.solutions.md]: `ALPHABET_SIZE` maybe?

File: algorithms/solutions/week21/duythinht.solutions.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    for (int i=0; i < ABC_SIZE; i++) {
+        if (chars_count[i] != 0) {
+            return false;
+        }
+    }
+    return true;
+}
+
+// This is stupid naming in C, don't want use this
+bool isAnagram(char* s, char* t) {
```
Comment (qcam) [inline algorithms/solutions/week21/duythinht.solutions.md]: bend your knees to the language!

Comment (huytd) [reply algorithms/solutions/week21/duythinht.solutions.md]: Ain't nobody got time for that? :yaomin:

File: algorithms/solutions/week21/duythinht.solutions.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    // Set default chars_count to zero array
+    memset(chars_count, 0, ABC_SIZE * sizeof(int));
+    
+    while (*s && *t) {
+        chars_count[*(s++)-'a']++; // Don't do this, sida đấy
+        chars_count[*(t++)-'a']--;
+    }
+    
+    // Avoid next step because of 2 str have difference length
+    if (*s || *t) {
```
Comment (huytd) [inline algorithms/solutions/week21/duythinht.solutions.md]: Oh, nice

File: algorithms/solutions/week21/duythinht.solutions.md (hunk 1 above)
Comment (duythinht) [reply algorithms/solutions/week21/duythinht.solutions.md]: Alphabet is too long dude :trollface:

**Pull request merged**